pyscript.core/test/display.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>PyScript Next</title>
+        <script>
+            addEventListener("py:all-done", ({ type }) => console.log(type));
+        </script>
+        <link rel="stylesheet" href="../dist/core.css">
+        <script type="module" src="../dist/core.js"></script>
+    </head>
+    <body>
+      <script type="py" worker async>
+          from pyscript import display
+          display('hello 1')
+
+          import js
+          import time
+          js.console.log('sleeping...')
+          time.sleep(2)
+          js.console.log('...done')
+      </script>
+      <p>hello 2</p>
+      <script type="py" worker async>
+          from pyscript import display
+          display('hello 3')
+      </script>
+    </body>
```

Comment:
this should become a proper integration test

Reply:
we could do a follow up but the feature is from *polyscript* so eventually tests should be done there, imho.

Reply:
actually, never mind, these probably should be done in both places because custom types events are not *polyscript* matter anymore ... that being said, we currently have `time` working in either sync or async scripts/elements ... not sure how to test this best as async and time don't really use async feature there. If you'd like to add a better test, please go ahead, thank you!